src/actions/ActionHelpers.ts

```diff
@@ -0,0 +1,664 @@
+// eslint-disable-next-line import/no-unresolved
+import { Frame, FrameType, IHasName, Layout, ListVariable, Page, StylekitChacterStylesController, StylekitColorsController, StylekitParagraphStylesController, Variable, VariableValue } from 'grafx-studio-actions';
+
+/**
+ * Whether the variable is the trigger for this action
+ * @param name optional variable name
+ * @returns true if it was the trigger, otherwise false
+ */
+export function isVariableTriggered(name?: string | Variable): boolean {
+    const changedVariable = triggers.changedVariable;
+
+    if (!changedVariable){
+        return false;
+    }
+
+    let variableName = name;
+
+    if (name && (name as Variable).name) {
+        variableName = (name as Variable).name;
+    }
+
+    if (typeof variableName !== 'string') {
+        throw new Error('variable argument should be of type "string" but was ' + typeof variableName);
```

Comment:
@Matthiee this will throw an error if variableName is undefined instead of saying "undefined"